src/nidmm/examples/nidmm_multi_point_measurement.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+
+import argparse
+import nidmm
+import sys
+
+parser = argparse.ArgumentParser(description='Performs a multi point measurement using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+parser.add_argument('-f', '--function', default='DC_VOLTS', choices=nidmm.Function.__members__.keys(), type=str.upper, help='Measurement function.')
+parser.add_argument('-r', '--range', default=10, type=float, help='Measurement range.')
+parser.add_argument('-d', '--digits', default=6.5, type=float, help='Digits of resolution for the measurement.')
+parser.add_argument('-s', '--samples', default=10, type=int, help='The number of measurements the DMM makes.')
+parser.add_argument('-to', '--timeout', default=-1, type=int, help='Specifies the maximum_time allowed for this function to complete in milliseconds.')
+parser.add_argument('-tc', '--trigger_count', default=1, type=int, help='Sets the number of triggers you want the DMM to receive before returning to the Idle state.')
```

Comment:
Command line argument flags usually use dash not underscore.
trigger-count

Reply:
fixed